README.rst

```diff
@@ -8,6 +8,9 @@ Usage
 
 Installation
 ------------
+1. Install node.js and Angular CLI as described in https://angular.io/guide/setup-local
```

Comment:
It would be a great help for if you could provide Dockerfile for this setup.

Reply:
Moved to #16